src/PrestaShopBundle/Controller/Api/ApiController.php

```diff
@@ -55,7 +56,7 @@ public function setLogger(LoggerInterface $logger)
      */
     protected $container;
 
-    public function setContainer(ContainerInterface $container)
+    public function setContainer(ContainerInterface $container = null)
```

Comment:
This change is needed to comply with Symfony controller contract

Reply:
Since we change the contract, it's a BC, you can't anymore pass null in public methods.

Reply:
If I am not wrong, this change is backward compatible 🤔 ?

Reply:
You're right, sometimes I read the red line as a new line -_-